.changeset/gentle-tomatoes-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/modal': patch
```

Comment:
I'm inclined to say this is breaking since it affects any existing implementations of initial focus consumers may be using.